scripts/update-key

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# Example run
+# ```
+# $ ./scripts/update-key b251d15e720672cbbe73626447a8458ffa1ab1413925cd51ef990e652d48318A
+# Update your sydent config file with the following:
+#
+# [crypto]
+# ed25519.signingkey = ed25519 0 slHRXnIGcsu+c2JkR6hFj/oasUE5Jc1R75kOZS1IMYo
+#
+# For reference, the public (verificiation) key is 53eNltXamSKdvxIgL3Tq4KMrwgR/sQA18xvwvxEYc4o
+# ````
+
+# Use this to update the signing key used in sydent configurations. This doesn't
+# change the key, only the way that it's encoded in the config file.
```

Comment:
Maybe this is clearer? I think it would make sense to include this above the examples.

```suggestion
# Run this script to convert the signing key used in a sydent configuration. This doesn't
# change the key, only the way that it's encoded in the config file.
```